### DISCUSSION TITLE
Read/Write a DICONDE file with Python

### DISCUSSION THREAD

**Discussion by RegalArtifex** [community/community#28783] [Other Feature Feedback, Questions, & Ideas]
I would like to read/write a DICONDE file using Python. I've tried Pydicom package too. It doesn't work either

**Comment by murilochaves** [comment]
For read, you can test:

`pip install pydicom`

```
import pydicom as dicom
import matplotlib.pylab as plt

# specify your image path
image_path = 'data/train_images/sample.dcm'
ds = dicom.dcmread(image_path)

plt.imshow(ds.pixel_array)
```
And to write... you need to assign PixelData a bytes string representation of the Image. Image is a numpy array and needs to be coverted to a bytes string.

`ds.PixelData = Image.tobytes()`
